esp/services/ws_access/ws_accessService.cpp

```diff
@@ -385,6 +354,46 @@ bool Cws_accessEx::getNewFileScopeNames(CLdapSecManager* secmgr, const char* nam
     return true;
 }
 
+//Parse a filescope spec (fs1::fs2::fs3) and populate the scopes array with each sub filespec (fs1, fs1::fs2, fs1::fs2::fs3).
+void Cws_accessEx::readFileScopesFromString(const char* spec, StringArray& scopes, bool append)
+{
+    if (isEmptyString(spec))
+        return;
+
+    StringBuffer newFileScope;
+    char* scopeStart = (char*) spec;
```

Comment:
Is there a reason these cannot be const char * rather than char *?  Safer not to remove the const.

Reply:
It was copied from the existing code. I change it now. 

Do you want me to retarget to 9.0.x?